### PR TITLE
fix(perf): Adjust copy in widget subtitles

### PR DESCRIPTION
**Pull request by k-fish**
### Summary
When the widgets aren't strictly in a predefined period we should show a blank term, this replaces it with 'period' for now.

### CODE REVIEW

File: static/app/views/performance/landing/widgets/widgets/histogramWidget.tsx [hunk 1]
```diff
@@ -61,7 +61,12 @@ export function HistogramWidget(props: Props) {
     <GenericPerformanceWidget<AreaDataType>
       {...props}
       Subtitle={() => (
-        <Subtitle>{t('Compared to last %s ', globalSelection.datetime.period)}</Subtitle>
+        <Subtitle>
+          {t(
+            'In the last %s ',
```
Comment (Zylphrex) [inline static/app/views/performance/landing/widgets/widgets/histogramWidget.tsx:66]: Changed from `Compared` to `In` intentional?

**Review by Zylphrex — APPROVED** [review]

File: static/app/views/performance/landing/widgets/widgets/histogramWidget.tsx [hunk 2]
```diff
@@ -61,7 +61,12 @@ export function HistogramWidget(props: Props) {
...
+            globalSelection.datetime.period ? globalSelection.datetime.period : 'period'
+          )}
```
Comment (evanpurkhiser) [inline static/app/views/performance/landing/widgets/widgets/histogramWidget.tsx:68]: the string `period` won't be translated. Should this actually be more like


```suggestion
          {globalSelection.datetime.period
            ? t('In the last %s ', globalSelection.datetime.period)
            : t('In the last period')}
```

File: static/app/views/performance/landing/widgets/widgets/singleFieldAreaWidget.tsx [hunk 3]
```diff
@@ -88,7 +88,12 @@ export function SingleFieldAreaWidget(props: Props) {
     <GenericPerformanceWidget<DataType>
       {...props}
       Subtitle={() => (
-        <Subtitle>{t('Compared to last %s ', globalSelection.datetime.period)}</Subtitle>
+        <Subtitle>
+          {t(
+            'Compared to last %s ',
+            globalSelection.datetime.period ? globalSelection.datetime.period : 'period'
+          )}
```
Comment (evanpurkhiser) [inline static/app/views/performance/landing/widgets/widgets/singleFieldAreaWidget.tsx:95]: same as above

File: static/app/views/performance/landing/widgets/widgets/histogramWidget.tsx (hunk 2 above)
Comment (k-fish) [reply static/app/views/performance/landing/widgets/widgets/histogramWidget.tsx:68]: Oh right 👍

**Pull request merged**